readthedocs/donate/templates/donate/promo_detail.html

```diff
@@ -32,10 +39,10 @@
 
 <table>
   <tr>
-    <th><strong>Day (UTC)</strong></th>
-    <th><strong>Views</strong></th> 
-    <th><strong>Clicks</strong></th>
-    <th><strong>CTR</strong></th>
+    <th><p class ="strong">Day (UTC)</p></th>
+    <th><p class ="strong">Views</p></th> 
+    <th><p class ="strong">Clicks</p></th>
+    <th><p class ="strong">CTR</p></th>
```

Comment:
My feedback to the original approach was that this didn't need to use `<strong>` tags on each cell. Instead of applying `<strong>` or `<p class="strong">` to each `<th>`, we should instead be applying a heavier weight to the `table tr th` css selector.